### DISCUSSION TITLE
Expansion for "cmi.anvil.colors"

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#6212] [Enhancement]
EDITED:
Please implement anvils as a "type" for the "cmi.colors.[type].[colorname/hex]" permissions.
Right now the only permission for it is "cmi.anvil.colors", which lets the player use all colors and font styles.

**Comment by mnfu** [comment]
I'd like to see this as well

**Comment by CloudeLecaw** [comment]
if added it would be nice to be able to see the setted colors also in /cmi colorlimits commands listed
